akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -1452,6 +1452,20 @@ trait FlowOps[+Out, +Mat] {
   def idleTimeout(timeout: FiniteDuration): Repr[Out] = via(new Timers.Idle[Out](timeout))
 
   /**
+    * If the time between emission of element and following backpressure notification exceed the provided timeout,
```

Comment:
"backpressure notification", is that a concept we have used anywhere else? does the user understand what it is?

I think we have used descriptions like "until downstream demand arrives"

What is the difference between this stage and using `keepAlive` followed by a `map` that throws when it sees the keep alive element?